components/org.wso2.carbon.identity.oauth.common/src/main/java/org/wso2/carbon/identity/oauth/common/OAuthConstants.java

```diff
@@ -227,9 +227,12 @@ public static SubjectType fromValue(String text) {
     public static final String ORG_ID = "org_id";
     public static final String IS_FAPI_CONFORMANT_APP = "isFAPIConformant";
     public static final String ENABLE_FAPI = "OAuth.OpenIDConnect.FAPI.EnableFAPIValidation";
-    public static final String IS_THIRD_PARTY_APP = "isThirdPartyApp";
-
+    public static final String ENABLE_DCR_FAPI_VALIDATION = "OAuth.DCRM.EnableFAPIEnforcement";
```

Comment:
Lets change constant name to match config name

```suggestion
    public static final String ENABLE_DCR_FAPI_ENFORCEMENT = "OAuth.DCRM.EnableFAPIEnforcement";
```

Reply:
changed